qiskit_optimization/optimizers/__init__.py

```diff
@@ -11,23 +11,20 @@
 # that they have been altered from the originals.
 
 """
-Optimizers (:mod:`qiskit_algorithms.optimizers`)
-================================================
+Optimizers (:mod:`qiskit_optimization.optimizers`)
+==================================================
 Classical Optimizers.
 
 This package contains a variety of classical optimizers and were designed for use by
-qiskit_algorithm's quantum variational algorithms, such as :class:`~qiskit_algorithms.VQE`.
+qiskit_optimization's quantum variational algorithms, such as
+:class:`~qiskit_optimization.minimum_eigensolvers.SamplingVQE`.
 Logically, these optimizers can be divided into two categories:
```

Comment:
> Logically, these optimizers can be divided into two categories:

Given that one of the two categories i.e. Global Optimizers has been removed by the changes here the (above) text really needs redoing.